src/mudlet-mapper.xml

```diff
@@ -222,40 +222,40 @@ end</script>
             <Trigger isActive="yes" isFolder="no" isTempTrigger="no" isMultiline="no" isPerlSlashGOption="no" isColorizerTrigger="no" isFilterTrigger="no" isSoundTrigger="no" isColorTrigger="no" isColorTriggerFg="no" isColorTriggerBg="no">
                 <name>Alertness (Achaea+Imperian)</name>
                 <script>mmp.alertness = mmp.alertness or {}
-
 local alertness = mmp.alertness
-
+ 
```

Comment:
Just a minor niggle but are these blank lines with just a space in them? Doesn't Git moan about these in some circumstances?